core/kits/assets/js/globals/global-select-behavior.js

```diff
@@ -17,9 +17,20 @@ export default class GlobalControlSelect extends Marionette.Behavior {
 	// This method exists because the UI elements are printed after controls are already rendered.
 	registerEvents() {
 		this.ui.globalPreviewsContainer.on( 'click', '.e-global__preview-item', ( event ) => this.applySavedGlobalValue( event.currentTarget.dataset.globalId ) );
-		this.ui.globalControlSelect.on( 'click', ( event ) => this.toggleSelect( event ) );
+
+		// TODO: Find better solution.
+		if ( this.ui.globalControlSelect ) {
```

Comment:
Why does `globalControlSelect` not always exist?

Reply:
I don't know its better to ask Udi i think.

Reply:
We can't ignore the problem. We have to fix it

Reply:
@udidol can you help?

Reply:
This is a problem we know about for a while now. 
I have tried to debug it and find why it happened and couldn't - I have no idea what causes this bug.
I agree we absolutely have to fix it.

Reply:
Remove this fix from this PR. We will try to resolve it on our side

Reply:
Handled: https://github.com/elementor/elementor/pull/11791/commits/9bdc1e71fd9955ff123ec9b4a9bf24bbaf13f05b